src/generator.rs

```diff
@@ -67,48 +75,81 @@ impl Generator {
                     .include_file(Self::TONIC_INCLUDE_FILE)
                     .compile(&protos, includes)?;
             }
+            Generator::Protoc { language, out_dir } => {
+                let mut protoc_cmd = std::process::Command::new(protoc);
```

Comment:
please use tokios version (which is `async`)

see: https://docs.rs/tokio/latest/tokio/process/struct.Command.html